src/stores.ts

```diff
@@ -115,16 +115,31 @@ interface State {
 // NOTE(sr):
 // 1. "package conditions" and "query" is a must
 // 2. we need the rego.v1 import because the Preview API has no "v1" flag
-const defaultRego = `package conditions
+let rego = new Map<string, string>();
+rego.set("orders", `package conditions
```

Comment:
I'd go with an object:
```suggestion
const rego = { orders: `package conditions
```